src/components/CurrentDirectors/CurrentDirectors.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div id="current-directors">
+
+    <v-card flat>
+      <!-- Header -->
+      <div class="header-container">
+        <v-icon color="app-dk-blue">mdi-account-multiple-plus</v-icon>
+        <label class="font-weight-bold pl-2">Current Directors</label>
+      </div>
+
+      <!-- Instructional Text -->
+      <div class="instructional-text pt-10 pb-15 px-4">
```

Comment:
We have found it easier to not use bottom margin/padding for spacing purposes. Instead, the element below this can add the necessary margin/padding.

This may be more of a convention (eg, either select upper or lower element to do the spacing), but if we pick one way then it's a lot cleaner and easier to update later.